migrations/versions/d1a281f6f2dd_.py

```diff
@@ -0,0 +1,51 @@
+"""empty message
+
+Revision ID: d1a281f6f2dd
+Revises: 5003b9acb1f4
+Create Date: 2024-12-13 13:28:45.942714
+
+"""
+import logging
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from yaptide.persistence.models import TaskModel
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1a281f6f2dd'
+down_revision = '5003b9acb1f4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('Task', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('sim_pid', sa.Integer(), nullable=False, server_default=sa.text("-1")))
```

Comment:
Why don't you assume these columns to be nullable ?

Reply:
This columns can be nullable, in addition i will move them to CeleryTaskModel because they are used only by celery simulation